mswindows/GRASS-Installer.nsi.tmpl

```diff
@@ -72,9 +63,9 @@ SetCompressorDictSize 128
 	!define DISPLAYED_NAME "GRASS GIS ${VERSION_NUMBER}-${BINARY_REVISION} (${PLATFORM})"
 	!define CHECK_INSTALL_NAME "GRASS GIS @GRASS_VERSION_MAJOR@@GRASS_VERSION_MINOR@"
 !else
-	!define INSTALLER_NAME "WinGRASS-${VERSION_NUMBER}-${SVN_REVISION}-${BINARY_REVISION}-Setup-${PLATFORM}.exe"
-	!define DISPLAYED_NAME "GRASS GIS ${VERSION_NUMBER}-${SVN_REVISION}-${BINARY_REVISION} (${PLATFORM})"
-	!define CHECK_INSTALL_NAME "GRASS GIS @GRASS_VERSION_MAJOR@@GRASS_VERSION_MINOR@ SVN"
+	!define INSTALLER_NAME "WinGRASS-${VERSION_NUMBER}-${GIT_REVISION}-${BINARY_REVISION}-Setup-${PLATFORM}.exe"
```

Comment:
@hellik What about removing $PLATFORM here? Do we need to promote `x86_64` in displayed/installer names?

Reply:
As we only provide 64bit binaries in the future,  I think we don't need to promote platform anymore.

Reply:
Done by 12850c4b5ea51faca02812df09b576051633f6da